README.md

```diff
@@ -36,19 +36,68 @@ scrape_configs:
     metrics_path: /export
     params:
       command: [check_load] # Run the check_load command.
+    relabel_configs:
+      - source_labels: [__address__]
+        target_label: __param_target
+      - source_labels: [__param_target]
+        target_label: instance
+      - target_label: __address__
+        replacement: 127.0.0.1:9275 # Nrpe exporter.
     static_configs:
       - targets: # Targets to run the specified command against.
         - '127.0.0.1:5666'
         - 'example.com:5666'
+
+  - job_name: nrpe_check_apt
+    honor_timestamps: true
+    params:
+      command: [check_apt]
+      ssl: [true]
+    scrape_interval: 3h # no need to poll this more frequently
+    scrape_timeout: 60s # this command could take a long time to execute
+    metrics_path: /export
+    scheme: http
+    follow_redirects: true
     relabel_configs:
       - source_labels: [__address__]
         target_label: __param_target
       - source_labels: [__param_target]
         target_label: instance
       - target_label: __address__
-        replacement: 127.0.0.1:9275 # Nrpe exporter.
+        replacement: nrpe:9275 # Nrpe exporter.
```

Comment:
Why this domain rather than localhost?

Reply:
Rather than using host networking, bridge (or other inter container communication) network is preferred as this service does not need to be visible outside the 'stack' ​.

I'll provide a bit more  documentation to clarify in next commit.

Reply:
This sounds like something relevant to your local setup rather than an average user, please leave it as-is.

Reply:
Will do. (This host name is a result of putting the prometheus-related containers in docker-compose file which has become typical - but is out of scope)